src/src/Animals/Creation.java

```diff
@@ -1,20 +1,38 @@
 package Animals;
 
+import java.sql.SQLOutput;
+import java.util.HashMap;
+import java.util.Map;
+
 public class Creation {
     public static void main(String[] args) {
-        Pet pet = new Pet("Nick", 5);
-        petCommands(pet);
-
-        Dog dog = new Dog("Guffi", 2, 5, "Grey", 11.5533);
-        dogCommands(dog);
+        Cat catBarsik = new Cat ("Barsik", 4, "Calm");
```

Comment:
It would be better to provide users with an opportunity to add data to create object, as well as decide what to add/update/delete in db for example. It must for course project